clients/admin-ui/src/pages/user-management/new.tsx

```diff
@@ -6,58 +6,42 @@ import {
   Heading,
 } from '@fidesui/react';
 import type { NextPage } from 'next';
-import { getSession } from 'next-auth/react';
+import Link from 'next/link';
 import React from 'react';
 
-import { wrapper } from '../../app/store';
+import ProtectedRoute from '../../features/auth/ProtectedRoute';
 import NavBar from '../../features/common/NavBar';
-import { assignToken, setUser } from '../../features/user/user.slice';
 import NewUserForm from '../../features/user-management/NewUserForm';
 
 const CreateNewUser: NextPage = () => (
-  <div>
-    <NavBar />
-    <main>
-      <Box px={9} py={10}>
-        <Heading fontSize='2xl' fontWeight='semibold'>
-          User Management
-          <Box mt={2} mb={7}>
-            <Breadcrumb fontWeight='medium' fontSize='sm'>
-              <BreadcrumbItem>
-                <BreadcrumbLink href='/user-management'>
-                  User Management
-                </BreadcrumbLink>
-              </BreadcrumbItem>
+  <ProtectedRoute>
```

Comment:
The `ProtectedRoute` component seems more elegant than the original `getServerSideProps`
solution.  Just wrap the JSX with 1 more component and it's being authenticated.

Reply:
Yep! To be fair, we can definitely improve on the `getServerSideProps` pattern, but I hadn't cleaned that up because I knew we were intending to rip it out.

I will say that one thing I noticed with this pattern is that it's easily possible to forget to authenticate a page, so we just need to either a) wrap all of the pages by default at the `App.tsx` level, or stay on top of that while developing. I'm not super concerned, since the requests wouldn't be authenticated anyhow, so even if we do forget to wrap a page it's not the end of the world – and we don't create pages that frequently, so the error surface area is low. But it's just another place where we could introduce an error.

To be fair, that same logic applied to the previous solution too, so I guess it's not a new problem. Something to think about...